x/committee/types/committee.go

```diff
@@ -13,32 +13,60 @@ import (
 
 const MaxCommitteeDescriptionLength int = 512
 
-// ------------------------------------------
-//				Committees
-// ------------------------------------------
+type TallyOption int
+
+const (
```

Comment:
Add comment to each of the tally options that describes them.

FPTH - Votes are tallied each block and the proposal passes as soon as the vote threshold is reached
Deadline - Votes are tallied exactly once, when the deadline time is reached

Reply:
https://github.com/Kava-Labs/kava/pull/899/commits/5ff4eeb6591dcf2040bb900af67fdfe6b2165c98